Измените структуру Person, чтобы она включала Address вместо строки A…

IlyasSV opened the pull request:
Задача 9: Анонимные поля
Создайте структуру Employee, которая будет включать анонимное поле типа Person, а также поле Position.+++


Задача 10: Использование анонимных полей
Инициализируйте несколько экземпляров структуры Employee и выведите их на экран.

Commit:
Измените структуру Person, чтобы она включала Address вместо строки Address.

valilhan commented on task1/main.go:
```diff
@@ -63,4 +63,20 @@ func main() {
+		},
+		Position: "1",
+	}
+	employee1.PrintEmployee()
```
В задачи сказано Инициализируйте несколько экземпляров. Таким образом их нужно собрать в один слайс и пройдясь по каждому элементу в слайсе вызвать функцию PrintEmployee

valilhan commented on model/employee.go:
```diff
@@ -0,0 +1,14 @@
+	Position string
+}
+
+func (e Employee) PrintEmployee() {
```
Можно было здесь использовать (e *Employee) где e была бы указателем на Employee

Commit:
1. В задачи сказано Инициализируйте несколько экземпляров. Таким образом их нужно собрать в один слайс и пройдясь по каждому элементу в слайсе вызвать функцию PrintEmployee
2. Можно было здесь использовать (e *Employee) где e была бы указателем на Employee
